### PR TITLE
MS19053: [CreateApp/EntityList] Move export selection button to bottom of the list

### PR DESCRIPTION
**Pull request by thoys**
https://highfidelity.manuscript.com/f/cases/19053/Add-Export-Selection-button-to-entity-list

## QA test

Run the following test rail tests on this PR:
- https://highfidelity.testrail.net/index.php?/cases/view/17785
- https://highfidelity.testrail.net/index.php?/cases/view/17767

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13607/HighFidelity-Beta-PR14136-66c533b.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13607/HighFidelity-Beta-PR14136-66c533b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13607/HighFidelity-Beta-Interface-PR14136-66c533b.dmg

File: scripts/system/html/js/entityList.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ const COMPARE_DESCENDING = function(a, b) {
 }
 
 // List of all entities
-var entities = []
+var entities = [];
```
Comment (dback2) [inline scripts/system/html/js/entityList.js]: You're finding all the missing semi-colons hehe ;)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13924/HighFidelity-Beta-PR14136-f188515.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13924/HighFidelity-Beta-Interface-PR14136-f188515.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13924/HighFidelity-Beta-PR14136-f188515.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13924/HighFidelity-Beta-Interface-PR14136-f188515.exe

**Comment by QliCknox** [comment]
Currently Testing

**Comment by QliCknox** [comment]
Pass @conklin94122 

Export Selection button works as expected and has moved to the bottom on the 'Entity List' window

**Pull request merged**